css/properties/margin-right.json

```diff
@@ -4,6 +4,7 @@
       "margin-right": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/margin-right",
+          "spec_url": "https://drafts.csswg.org/css-flexbox-1/#item-margins",
```

Comment:
```suggestion
          "spec_url": "https://drafts.csswg.org/css-box/#propdef-margin-right",
```

Reply:
Not https://drafts.csswg.org/css-box/#margin-physical?

Reply:
```suggestion
          "spec_url": "https://drafts.csswg.org/css-box/#margin-physical",
```